features/old/services/multiservice.feature

```diff
@@ -31,14 +30,49 @@ Feature: Multiservice feature
   @javascript
   Scenario: Create new product: Fail scenario error message
     Given I am logged in as provider "foo.3scale.localhost"
-    And provider "foo.3scale.localhost" has "multiple_services" switch allowed
-    And service discovery is not enabled
+      And provider "foo.3scale.localhost" has "multiple_services" switch allowed
+      And service discovery is not enabled
     When I am on the provider dashboard
-    And I follow "Create Product"
-    And I fill in "Name" with "Invalid API"
-    And I fill in "System name" with "I am using spaces"
-    And I press "Create Product"
-    Then I should see the flash message "System name invalid. Only ASCII letters, numbers, dashes and underscores are allowed."
+      And I follow "Create Product"
+      And I fill in "Name" with "Invalid API"
+      And I fill in "System name" with "I am using spaces"
+      And I press "Create Product"
+    Then I should see the flash message "Product could not be created"
+
+    @javascript
+    Scenario: Create new product: with blank product name
+      Given I am logged in as provider "foo.3scale.localhost"
+        And provider "foo.3scale.localhost" has "multiple_services" switch allowed
+        And service discovery is not enabled
+      When I am on the provider dashboard
+        And I follow "Create Product"
+        And I fill in "System name" with "Less fancy API"
+        And I press "Create Product"
+      Then I should see "Can't be blank"
```

Comment:
What can't be blank? We should assert specific field errors.

Reply:
it shows error like this only , but scenario is complete

![image](https://user-images.githubusercontent.com/84672731/233578952-918f1730-f30b-4cd7-b834-97f5d292df00.png)

Reply:
Yes, we should assert specific field errors, not generic content on screen. For example:
```suggestion
      Then I should see Name can't be blank
```

Reply:
![image](https://user-images.githubusercontent.com/84672731/233913615-22e004d0-f7cb-40d9-866b-4a22dcf4fbac.png)